playbook/app/pb_kits/playbook/pb_image/_image.jsx

```diff
@@ -30,7 +30,9 @@ const Image = (props: ImageProps) => {
 
   const ariaProps = buildAriaProps(aria)
   const classes = classnames(
-    buildCss('pb_image_kit lazyload blur_up', size),
+    buildCss('pb_image_kit', size),
+    'lazyload',
+    'blur_up',
```

Comment:
These were out of order and now the Rails and React generated class names are consistent.